File: Assignment8/program1.c
////////////////////////////////////////////////////////////////
//
//  File Name   : program1.c
//  Description : Accept a Integer and returns Area of Circle
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 17/05/2025
//
////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////
// 
//  Function Name : CircleArea
//  Description   : Returns Area of Circle for input float
//  Input         : Float
//  Output        : Double
//
////////////////////////////////////////////////////////////////

double CircleArea(float fRadius)
{
     double Area = 3.14 * fRadius * fRadius;
     return Area;
}
int main()
{
    float fValue = 0.0;
    double dRet = 0.0;
    
    printf("Enter radius");
    scanf("%f",&fValue);
    
    dRet = CircleArea(fValue);
    
    printf("Area of circle : %lf",dRet);
    
    return  0;
}

File: Assignment13/program1.java
////////////////////////////////////////////////////////////////
//
//  File Name   : program1.java
//  Description : Accept a Integer and Print 'Character' Pattern
//  Author      : Akhilesh.P.Sonavane.
//  Date        : 22/05/2025
//
////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////
// 
//  Class Name    : Pattern
//  Function Name : Display
//  Description   : Inputs a Integer and returns 'Character' Pattern
//  Input         : Integer
//  Output        : Character
//
////////////////////////////////////////////////////////////////

class Pattern 
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;
        char ch = '\0';

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1, ch = 'A'; j <= iCol; j++,ch++)
            {
                System.out.print(ch+"\t");
            }
            System.out.print("\n\n");
        }
    }
}
public class program1
{
    public static void main(String[] args) 
    {
        int iValue1 = 0;
        int iValue2 = 0;

        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter Number of Coloumns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1,iValue2);

        sobj.close();
    }
}
